subprojects/docs/src/main/resources/header.html

```diff
@@ -95,92 +95,78 @@
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#upgrading-gradle" aria-expanded="false" aria-controls="upgrading-gradle">Upgrading Gradle</a>
+            <li><a href="/userguide/installation.html">Installation</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#upgrading-gradle" aria-expanded="false" aria-controls="upgrading-gradle">Upgrading &amp; Migrating</a>
                 <ul id="upgrading-gradle">
-                    <li><a href="/userguide/upgrading_version_4.html">version 4.x to 5.0</a></li>
-                </ul>
-            </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#migrating-to-gradle" aria-expanded="false" aria-controls="migrating-to-gradle">Migrating to Gradle</a>
-                <ul id="migrating-to-gradle">
+                    <li><a href="/userguide/upgrading_version_4.html">Upgrading from Gradle 4.x to 5.0</a></li>
                     <li><a href="https://guides.gradle.org/migrating-from-maven/">Migrating From Maven</a></li>
                 </ul>
             </li>
-            <li id="tutorials"><a href="https://guides.gradle.org/#getting-started">Getting Started Guides</a></li>
-            <li id="third-party-integration"><a href="/userguide/third_party_integration.html">Gradle &amp; Third-party Tools</a></li>
+            <li id="tutorials"><a href="https://guides.gradle.org/#getting-started">Tutorials</a></li>
         </ul>
 
         <h3 id="reference">Reference</h3>
         <ul>
-            <li><a href="https://gradle.github.io/kotlin-dsl-docs/api/" target="_blank">Kotlin DSL Reference</a></li>
-            <li><a href="/dsl/">Groovy DSL Reference</a></li>
-            <li><a href="/javadoc/">Gradle API Javadoc</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#gradle-api" aria-expanded="false" aria-controls="gradle-api">Gradle API</a>
+                <ul id="gradle-api">
+                    <li><a href="/javadoc/">Javadoc</a></li>
+                    <li><a href="/dsl/">Groovy DSL</a></li>
+                    <li><a href="https://gradle.github.io/kotlin-dsl-docs/api/" target="_blank">Kotlin DSL</a></li>
+                </ul>
+            </li>
             <li><a href="/userguide/command_line_interface.html">Command-Line Interface</a></li>
             <li><a href="/userguide/plugin_reference.html">Core Plugins</a></li>
+            <li id="third-party-integration"><a href="/userguide/third_party_integration.html">Gradle &amp; Third-party Tools</a></li>
             <li><a href="https://docs.gradle.com/build-scan-plugin">Build Scan Plugin User Manual</a></li>
         </ul>
 
-        <h3 id="getting-started">Managing Gradle Projects</h3>
+        <h3 id="getting-started">Running Gradle Builds</h3>
         <ul>
-            <li><a href="https://guides.gradle.org/creating-new-gradle-builds/">Creating New Gradle Builds</a></li>
```

Comment:
I want to avoid unnecessary links to Guides where possible. This is now linked from the very first chapter of the Authoring Gradle Builds part. Also, this is more about authoring builds rather than running/managing them.